src/routes/tools/extreme-precipitation/_ExploreData.svelte

```diff
@@ -239,6 +265,7 @@
       units="{$climvar.units.imperial}"
       label="{indicatorLabel}"
       dataSource="{$titles.join(', ')}"
+      niceMax="{['intensity'].includes($indicator.id) ? 5 : 10}"
```

Comment:
It would be simpler just to do: `$indicator.id === 'intensity' ? 5 : 10` right?

Reply:
fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/206/commits/87be8c69e17888e595279e39a0a988f1c854e36a